qutebrowser/browser/webkit/downloads.py

```diff
@@ -757,7 +784,7 @@ def get(self, url, **kwargs):
             **kwargs: passed to get_request().
 
         Return:
-            If the download could start immediately, (fileobj/filename given),
+            If the download could start immediately, (target given),
             the created DownloadItem.
 
             If not, None.
```

Comment:
Correct, with the question removed it should always return the created `DownloadItem`.